src/style/__tests__/colors.spec.js

```diff
@@ -123,4 +123,62 @@ describe('style/Colors', () => {
       expect(palette).toEqual(['#661A66', '#8F248F', '#B82EB7', '#D148D1', '#DB71DB', '#E699E6', '#F0C2F0', '#FFE5FF']);
     });
   });
+
+  describe('generateTokens(...)', () => {
+    it('should generate design tokens from dark color for light theme', () => {
+      const chosenColor = '#860D86';
+      expect(uut.isDark(chosenColor)).toEqual(true);
+      expect(uut.generateTokens(chosenColor, 'light')).toEqual({
+        $backgroundPrimaryHeavy: chosenColor,
+        $backgroundPrimaryLight: '#FFFAFF',
+        $backgroundPrimaryMedium: '#FACCFA',
+        $iconPrimary: chosenColor,
+        $iconPrimaryLight: '#F16FF1',
+        $outlinePrimary: chosenColor,
+        $textPrimary: chosenColor
+      });
```

Comment:
These returned colors don't really match the ones in [figma](https://www.figma.com/file/Krv1pLl7kq2L52vMRwd498/Guidelines?node-id=11032%3A571101). I assume `generatePalette()` would need fixing here.

Reply:
Actually I think the fix should be in the figma board, I'll take it with the relevant UX.